libs/database/src/lib/entities/media.entity.ts

```diff
@@ -114,6 +114,22 @@ export class Media extends Model<Media> {
   @Column(BIGINT.UNSIGNED)
   modelId: number;
 
+  get modelName() {
+    return this.modelType === Project.LARAVEL_TYPE
+      ? "project"
+      : this.modelType === Nursery.LARAVEL_TYPE
+      ? "nursery"
+      : this.modelType === Site.LARAVEL_TYPE
+      ? "site"
+      : this.modelType === NurseryReport.LARAVEL_TYPE
+      ? "nursery-report"
+      : this.modelType === ProjectReport.LARAVEL_TYPE
+      ? "project-report"
+      : this.modelType === SiteReport.LARAVEL_TYPE
+      ? "site-report"
```

Comment:
I'm not sure what these are being used for yet (maybe something in the FE?), but in general, v3 is trying to be consistent about how model names are specified. See `entities.ts` - v3 prefers plural, camelCased names.